frontend/static/index.html

```diff
@@ -7,12 +7,6 @@
     </head>
     <body>
         <div id="root"></div>
-        <script crossorigin src="https://unpkg.com/react@16/umd/react.development.js"></script>
```

Comment:
hm. I think there is a way to tell Webpack that React is available as an external global, so that it can be preloaded from CDN? I don't know if the tree-shaking stuff makes the effect of loading it ourselves pointless...

(nonblocking)

Reply:
I did this to get unminified errors in development. As far as I know, the previous way using `externals` is the right way to load from CDN